src/adagrams.js

```diff
@@ -1,15 +1,121 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const letters = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A',
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D',
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E',
+    'F', 'F',
+    'G', 'G', 'G',
+    'H', 'H',
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I',
+    'J',
+    'K',
+    'L', 'L', 'L', 'L',
+    'M', 'M',
+    'N', 'N', 'N', 'N', 'N', 'N',
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O',
+    'P', 'P',
+    'Q',
+    'R', 'R', 'R', 'R', 'R', 'R',
+    'S', 'S', 'S', 'S',
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U',
+    'V', 'V',
+    'W', 'W',
+    'X',
+    'Y', 'Y',
+    'Z'
+];
+  let hand = [];
+  let letters2 = [...letters];
```

Comment:
Use const instead of let since neither variables are reassigned. A general rule of thumb for me when writing javascript is to use const and only refactor to let if the JS interpreter throws an exception when I try to reassign a variable.

Also prefer a more descriptive variable name like `lettersCopy`